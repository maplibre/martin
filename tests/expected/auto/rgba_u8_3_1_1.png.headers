
content-length: 294
vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
content-encoding: gzip
etag: "183196322924765665630178019653551670231"
content-type: image/png


