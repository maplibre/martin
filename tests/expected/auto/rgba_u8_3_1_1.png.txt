tests/output/auto/rgba_u8_3_1_1.png: PNG image data, 256 x 256, 8-bit/color RGBA, non-interlaced
